src/components/SpaceProposalVotesListItem.vue

```diff
@@ -114,3 +117,11 @@ const { formatCompactNumber } = useIntl();
     </div>
   </div>
 </template>
+
+<style scoped>
```

Comment:
Why do we need this, should we make it global maybe?

Reply:
We have some more tippy styling in style.scss

Reply:
ah took it from example https://vue-tippy.netlify.app/themes#creating-a-theme